Notebooks/README.md

```diff
@@ -0,0 +1,3 @@
+A place to store [Jupyter Notebook](https://jupyter.org/) files of ongoing studies.
```

Comment:
Suggestion/question: Can we maintain the patterns here for naming folders ? 😬 
Meaning, Is there a reason why is this different from the rest ? Didn't find anything on the docs